components/backup/src/endpoint.rs

```diff
@@ -450,20 +442,31 @@ impl BackupRange {
             return Ok(statistics);
         }
         let mut batch = vec![];
+        let cur_ts = ttl_current_ts();
```

Comment:
Nit: Suggest use less abbreviation to make codes more readable.
```suggestion
        let current_ts = ttl_current_ts();
```

Reply:
done